src/preact/base-element.js

```diff
@@ -324,7 +325,7 @@ export class PreactBaseElement extends AMP.BaseElement {
       } else {
         const container = doc.createElement('i-amphtml-c');
         this.container_ = container;
-        this.applyFillContent(container);
```

Comment:
Ok. This is a good track item: we really want to hollow out `CustomEleemnt` and `BaseElement` APIs. Sort of - everything that can become a static util - we should just make a static util, e.g. in a `dom.js` or a similar package.